Car_owners/car_owners/prod_settings.py

```diff
@@ -63,14 +65,15 @@
 # Database
 # https://docs.djangoproject.com/en/4.1/ref/settings/#databases
 
```

Comment:
prod and dev settings have much replicated code. Either create a common_settings file, or treat prod as base, import them to dev_settings and just overwrite the values you want to be different